crates/gitbutler-stack/src/patch_reference.rs

```diff
@@ -25,28 +25,13 @@ pub struct Branch {
     /// An identifier for a review unit at a forge (eg. GitHub Pull Request number).
     /// None if is no review unit, eg. no Pull Request has been created.
     #[serde(default)]
-    pub forge_id: Option<ForgeIdentifier>,
+    pub pull_request_id: Option<i32>,
```

Comment:
Perhaps update the docs above - is it a PR number or a PR id? The previous type had PR number. The GitHub api has both... and i always mix them up. I think the PR number needs a repo as well, while the PR ID is more globally unique or something. 

Also - you probably want `useize` not i32